src/command_templates.rs

```diff
@@ -0,0 +1,241 @@
+                    .ok_or(anyhow!("No uint found"))?;
+                if uint_match.start() != 0 || uint_match.end() != input_decomposed[input_idx].len()
+                {
+                    return Err(anyhow!("Uint must be the whole word"));
+                }
+                let mut uint_match = uint_match.as_str();
+                if uint_match.contains("</div>") {
+                    uint_match = uint_match.split("</div>").collect::<Vec<&str>>()[0];
+                }
+                let uint = U256::from_dec_str(uint_match).unwrap();
+                template_vals.push(TemplateValue::Uint(uint));
+            }
+            "{int}" => {
+                let int_match = Regex::new(INT_REGEX)
+                    .unwrap()
+                    .find(input_decomposed[input_idx])
+                    .ok_or(anyhow!("No int found"))?;
+                if int_match.start() != 0 || int_match.end() != input_decomposed[input_idx].len() {
+                    return Err(anyhow!("Int must be the whole word"));
+                }
+                let mut int_match = int_match.as_str();
+                if int_match.contains("</div>") {
+                    int_match = int_match.split("</div>").collect::<Vec<&str>>()[0];
+                }
+                let int = I256::from_dec_str(int_match).unwrap();
+                template_vals.push(TemplateValue::Int(int));
+            }
+            "{decimals}" => {
+                let decimals_match = Regex::new(DECIMALS_REGEX)
+                    .unwrap()
+                    .find(input_decomposed[input_idx])
+                    .ok_or(anyhow!("No decimals found"))?;
+                if decimals_match.start() != 0
+                    || decimals_match.end() != input_decomposed[input_idx].len()
+                {
+                    return Err(anyhow!("Decimals must be the whole word"));
+                }
+                let mut decimals = decimals_match.as_str().to_string();
+                if decimals.contains("</div>") {
+                    decimals = decimals.split("</div>").collect::<Vec<&str>>()[0].to_string();
+                }
+                template_vals.push(TemplateValue::Decimals(decimals));
+            }
+            "{ethAddr}" => {
+                let address_match = Regex::new(ETH_ADDR_REGEX)
+                    .unwrap()
+                    .find(input_decomposed[input_idx])
+                    .ok_or(anyhow!("No address found"))?;
+                if address_match.start() != 0 {
+                    return Err(anyhow!("Address must be the whole word"));
+                }
+                let address = address_match.as_str().parse::<Address>().unwrap();
+                template_vals.push(TemplateValue::EthAddr(address));
+            }
+            _ => {} // Skip unknown placeholders
+        }
+    }
+
+    Ok(template_vals)
+}
+
+// Generated by Github Copilot!
+pub fn uint_to_decimal_string(uint: u128, decimal: usize) -> String {
```

Comment:
Can you check if this functions exists in converters.rs if not can you shift it there?